crates/walrus-sui/src/client/retry_client/retriable_rpc_client.rs

```diff
@@ -554,16 +573,9 @@ impl RetriableClientError {
             {
                 true
             }
-            _ => self.is_failure_window_exceeded(last_success, num_failures),
+            _ => false,
         }
     }
-
-    /// Returns `true` if the failure window has been exceeded. Failure window is exceeded if
-    /// the number of failures exceeds `MAX_FAILURES` and if the last successful RPC call was
-    /// more than `FAILURE_WINDOW` minutes ago.
-    fn is_failure_window_exceeded(&self, last_success: Instant, num_failures: usize) -> bool {
```

Comment:
Note: this part is now moved to FallbackClient so that it can be checked against config.